posts/2023-07-13-Rust-1.71.0.md

```diff
@@ -0,0 +1,115 @@
+
+If you have a previous version of Rust installed via rustup, you can get 1.71.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.71.0](https://github.com/rust-lang/rust/releases/tag/1.71.0) on GitHub.
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.71.0 stable
+
+### C-unwind ABI
+
+1.71.0 stabilizes `C-unwind` (and other `-unwind` suffixed ABI variants[^1]).
+
+The behavior for unforced unwinding (the typical case) is specified in [this
+table from the RFC which proposed this feature][rfc-table]. To summarize:
+
+Each ABI is mostly equivalent to the same ABI without `-unwind`, except that
+the behavior is defined to be safe when an unwinding operation (`panic` or C++
+style exception) crosses the ABI boundary. For `panic=unwind`, this is a valid
+way to let exceptions from one language unwind the stack in another language
+without terminating the process (as long as the exception is caught in the same
+language from which it originated); for `panic=abort`, this will typically
+abort the process immediately.
+
+For this initial stabilization, *no change* is made to the existing ABIs (e.g.
+`"C"`). The existing ABIs have soundness holes around unwinding, though, which
+will be fixed when the behavior described in the RFC is stabilized. This change
+will happen in a future Rust release, and users are encouraged to start using
+the new unwind variants in their code to remain future proof if they wish to
+unwind across the ABI boundary.
+
+### Debugger visualization attributes
+
+1.71.0 stabilizes support for a new attribute, `#[debug_visualizer(natvis_file
+= "...")]` and `#[debug_visualizer(gdb_script_file = "...")]`, which allows
+embedding Natviz descriptions and GDB scripts into Rust libraries to
+improve debugger output when inspecting data structures created by those
+libraries. Rust itself has packaged similar scripts for some time for the
+standard library, but this feature makes it possible for library authors to
+provide a similar experience to end users.
+
+See the [reference](https://doc.rust-lang.org/nightly/reference/attributes/debugger.html#the-debugger_visualizer-attribute)
+for details on usage.
+
+### Const-initialized thread locals
+
+Rust 1.59.0 stabilized `const` initialized thread local support in the standard
+library, which allows for more optimal code generation. However, until now this
+feature was missed in release notes and
```

Comment:
I think it's worth noting that this is macro specific syntax and unrelated to the unstable inline_const feature.